build.gradle

```diff
@@ -222,6 +222,7 @@ project(':iceberg-core') {
     }
 
     testImplementation "org.xerial:sqlite-jdbc"
+    testImplementation "org.apache.commons:commons-lang3"
```

Comment:
What is this used for? We don't usually depend on Apache commons stuff.

Reply:
Just for the test cases, it has an easy method for generating random strings of specific lengths. https://github.com/apache/iceberg/pull/3966/files#diff-2f797a408a34d59d8b0d0a0502cf96af670e693be34f18768579ba85b6fdb45dR216

Reply:
You might want to just update our internal `RandomUtil` class instead. It's pretty close to doing what you want: https://github.com/apache/iceberg/blob/master/api/src/test/java/org/apache/iceberg/util/RandomUtil.java#L200-L209

Reply:
Are you ok with me changing the RandomUtil Class to actually do unicode? I think that's what we generally need here

Reply:
actually i'll just use the internal class for now. work on that later